### PR TITLE
Use the name of the spec file when install locally.

### PR DESCRIPTION
**Pull request by floitsch**

### CODE REVIEW

File: pkg/tpkg/manager.go [hunk 1]
```diff
@@ -367,12 +367,33 @@ func (m *ProjectPkgManager) InstallLocalPkg(ctx context.Context, name string, pa
...
+				if !doesPathExist(target_spec) {
+					return "", m.ui.ReportError("Missing package.yaml in '%s'", path)
+				}
+				return "", m.ui.ReportError("Package.yaml in '%s' is not a file", path)
+			}
+			return "", m.ui.ReportError("Package.yaml in '%s' is not a file: %v", path, err)
+		}
+		spec, err := ReadSpec(target_spec, m.ui)
+		if err != nil {
+			return "", m.ui.ReportError("Couldn't read spec file at '%s': %v", target_spec, err)
```
Comment (kasperl) [inline pkg/tpkg/manager.go:384]: Sometimes you call it 'spec file' and sometimes package.yaml. Maybe just stick with package.yaml in the error mesages?

File: pkg/tpkg/manager.go [hunk 2]
```diff
@@ -367,12 +367,33 @@ func (m *ProjectPkgManager) InstallLocalPkg(ctx context.Context, name string, pa
...
-	if !isValidName(name) {
-		return "", m.ui.ReportError("Invalid name: '%s'", name)
+	if name == "" {
+		target_spec := filepath.Join(abs, "package.yaml")
+		if isFile, err := isFile(target_spec); !isFile || err != nil {
+			if err == nil {
+				if !doesPathExist(target_spec) {
+					return "", m.ui.ReportError("Missing package.yaml in '%s'", path)
+				}
+				return "", m.ui.ReportError("Package.yaml in '%s' is not a file", path)
```
Comment (kasperl) [inline pkg/tpkg/manager.go:378]: I'd probably find a way to avoid capitalizing the filename here (Package.yaml) here and below.

**Review by kasperl — APPROVED** [review]
Looks good to me.

File: pkg/tpkg/manager.go (hunk 2 above)
Comment (floitsch) [reply pkg/tpkg/manager.go]: Done, but not super happy with the new error message.
Will try to come up with a better one for a future PR.

**Pull request merged**